core/src/snowcli/config.py

```diff
@@ -74,11 +74,43 @@ def add_connection(self, name: str, parameters: dict):
         self._find_section("connections").add(name, parameters)
         self._dump_config()
 
+    # TODO: think about extracting plugin specific methods to another layer
```

Comment:
IMO we have to think about moving config helpers for plugin somewhere outside CliConfigManager.
The same about connections.